peachjam/templates/peachjam/_document_table.html

```diff
@@ -1,9 +1,10 @@
 {% load i18n peachjam %}
-<div class="table-responsive">
+<div class="table-responsive" id="doc-table">
   <table class="doc-table">
     <thead>
       <tr>
-        <th scope="col" class="cell-title">{% trans 'Title' %}</th>
+        <th></th>
+        <th scope="col" class="w-75">{% trans 'Title' %}</th>
```

Comment:
the new `doc-table` class handles column spacing, let me check exactly what we need to do here.